mpas_analysis/default.cfg

```diff
@@ -1509,9 +1507,9 @@ colormapNameDifference = balance
 # whether the colormap is indexed or continuous
 colormapTypeDifference = indexed
 # color indices into colormapName for filled contours
-colormapIndicesDifference = numpy.array(numpy.linspace(0, 255, 9), int)
+colormapIndicesDifference = numpy.array(numpy.linspace(0, 255, 12), int)
```

Comment:
@xylar : the only thing I would change is the number of color indices: for extended colorbars, we want one *more* index than the number of color levels, not one less. If we change the above to `numpy.array(numpy.linspace(0, 255, 14), int)`, we'll have the two triangles for upper and lower bounds in two different colors.

Reply:
```suggestion
colormapIndicesDifference = numpy.array(numpy.linspace(0, 255, 14), int)
```

Reply:
Great, thanks @milenaveneziani.  That's an excellent point.